src/skillsaw/rules/docs/content-broken-external-reference.md

```
@@ -0,0 +1,284 @@
+`skillsaw badge` output becomes a function of whether a third-party URL
+404s today.
+
+## The operator's gate
+
+The rule's `enabled` state is decided by the linted repository's
+`.skillsaw.yaml` — which skillsaw's [threat model](https://github.com/stbenjam/skillsaw/blob/main/THREAT_MODEL.md)
+treats as untrusted content. Whether skillsaw may touch the network at
+all is decided by the operator instead:
+
+```console
+$ skillsaw lint . --no-network     # also: SKILLSAW_NO_NETWORK=1
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)